JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -439,11 +447,11 @@ - (UICollectionViewCell *)collectionView:(JSQMessagesCollectionView *)collection
 {
     id<JSQMessageData> messageItem = [collectionView.dataSource collectionView:collectionView messageDataForItemAtIndexPath:indexPath];
     NSParameterAssert(messageItem != nil);
-
+    
     NSString *messageSenderId = [messageItem senderId];
     NSParameterAssert(messageSenderId != nil);
```

Comment:
sorry, i might have been unclear with my previous comment :smile: 

let's remove this like you had it before since `isOutgoingMessage:` takes care of this